bin/compile

```diff
@@ -37,8 +37,7 @@ getEncoding () {
   echo $encoding
 }
 
-
-if [ -n $LANG ]; then
+if [[ -n $LANG ]]; then
```

Comment:
I'm partial to `if [ -n "$LANG" ]; then` (the quotes are important: -n "" and -n without an argument mean different things), for "is-set" checks, but this works as well.